hazelcast/src/test/java/com/hazelcast/internal/diagnostics/DistributedDatastructuresMetricsTest.java

```diff
@@ -90,6 +96,20 @@ public void testMap() {
         assertHasStatsEventually(MAP_NAME, "map.");
     }
 
+    @Test
+    public void testCache() {
+        final ICache<Object, Object> cache = hz.getCacheManager().getCache(CACHE_NAME);
+
+        Random random = new Random();
+        for (int i = 0; i < EVENT_COUNTER; i++) {
+            int key = random.nextInt(Integer.MAX_VALUE);
+            cache.putAsync(key, 23);
+            cache.removeAsync(key);
+        }
+
+        assertHasStatsEventually("/hz/" + CACHE_NAME, "cache.");
```

Comment:
minor: could also use `CacheUtil.getDistributedObjectName(CACHE_NAME)` instead of string concatenation

Reply:
Thanks, replaced with `CacheUtil`.